pkg/pipelines/bootstrap.go

```diff
@@ -18,13 +18,16 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+const (
+	DockerSecretName     = "regcred"
+	SaName               = "pipeline"
+	RoleName             = "pipelines-service-role"
+	RoleBindingName      = "pipelines-service-role-binding"
+	DevRoleBindingName   = "pipeline-edit-dev"
+	StageRoleBindingName = "pipeline-edit-stage"
+)
+
```

Comment:
Capitalize them only if they are exported.

Reply:
Also `pipeline-service-role` and `pipeline-service-role-binding` might fit better, given the rest of the names.